src/devices/Lcm1602a1/HD44780.cs

```diff
@@ -0,0 +1,559 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Gpio.Drivers;
+using System.Diagnostics;
+using System.Drawing;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+namespace Iot.Device.CharacterLCD
+{
+    /// <summary>
+    /// Supports LCD character displays compatible with the HD44780 LCD controller/driver.
+    /// Also supports serial interface adapters such as the MCP23008.
+    /// </summary>
+    /// <remarks>
+    /// The Hitatchi HD44780 was released in 1987 and set the standard for LCD controllers. Hitatchi does not make this chipset anymore, but
```

Comment:
I guess we should probably edit the README file to show this info instead, and say that we now support the actual chip controller, instead of just a single lcd device.